lib/chef/resource/windows_audit_policy.rb

```diff
@@ -0,0 +1,226 @@
+                     "Logon",
+                     "Logoff",
+                     "Account Lockout",
+                     "IPsec Main Mode",
+                     "IPsec Quick Mode",
+                     "IPsec Extended Mode",
+                     "Special Logon",
+                     "Other Logon/Logoff Events",
+                     "Network Policy Server",
+                     "User / Device Claims",
+                     "Group Membership",
+                     "File System",
+                     "Registry",
+                     "Kernel Object",
+                     "SAM",
+                     "Certification Services",
+                     "Application Generated",
+                     "Handle Manipulation",
+                     "File Share",
+                     "Filtering Platform Packet Drop",
+                     "Filtering Platform Connection",
+                     "Other Object Access Events",
+                     "Detailed File Share",
+                     "Removable Storage",
+                     "Central Policy Staging",
+                     "Sensitive Privilege Use",
+                     "Non Sensitive Privilege Use",
+                     "Other Privilege Use Events",
+                     "Process Creation",
+                     "Process Termination",
+                     "DPAPI Activity",
+                     "RPC Events",
+                     "Plug and Play Events",
+                     "Token Right Adjusted Events",
+                     "Audit Policy Change",
+                     "Authentication Policy Change",
+                     "Authorization Policy Change",
+                     "MPSSVC Rule-Level Policy Change",
+                     "Filtering Platform Policy Change",
+                     "Other Policy Change Events",
+                     "User Account Management",
+                     "Computer Account Management",
+                     "Security Group Management",
+                     "Distribution Group Management",
+                     "Application Group Management",
+                     "Other Account Management Events",
+                     "Directory Service Access",
+                     "Directory Service Changes",
+                     "Directory Service Replication",
+                     "Detailed Directory Service Replication",
+                     "Credential Validation",
+                     "Kerberos Service Ticket Operations",
+                     "Other Account Logon Events",
+                     "Kerberos Authentication Service",
+                    ]
+      resource_name :windows_audit_policy
+
+      description "The windows_audit_policy resource allows for configuring system and per-user Windows advanced audit policy settings."
+
+      examples <<~DOC
+      **Set Logon and Logoff policy to "Success and Failure"**:
+
+      ```ruby
+      windows_audit_policy "Set Audit Policy for "Logon and Logoff" actions to "Success and Failure" do
```

Comment:
This is not valid ruby code.
```suggestion
      windows_audit_policy "Set Audit Policy for 'Logon and Logoff' actions to 'Success and Failure'" do
```